patches/sequelize+6.13.0-handleDistincts.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/node_modules/sequelize/dist/lib/dialects/abstract/query-generator.js b/node_modules/sequelize/dist/lib/dialects/abstract/query-generator.js
```

Comment:
we had a fork of sequelize 5 with 2 fixes.

one of them is apparently no longer needed in sequelize 6, and this one I prefered to implement using patch-packages as it will be easier to later upgrade sequelize.